admin/src/graphql/allCommunities.js

```diff
@@ -0,0 +1,29 @@
+import gql from 'graphql-tag'
+
+export const allCommunities = gql`
+  query {
+    allCommunities {
+      foreign
+      url
+      publicKey
+      uuid
+      authenticatedAt
+      name
+      description
+      gmsApiKey
+      creationDate
+      createdAt
+      updatedAt
+      federatedCommunities {
+        id
+        apiVersion
+        endPoint
+        lastAnnouncedAt
+        verifiedAt
+        lastErrorAt
+        createdAt
+        updatedAt
+      }
+    }
```

Comment:
on my opinion causes this query the main problem of the following mismatching read/write-accesses and their mapping efforts or problems with `communities`- and `federatedCommunities`-Entities.

As you correctly recognize that in database there could exist `federatedCommunities`-Entries without a matching `communities`-entity. So how should this query here treat this constellation having no surrounding community as an anker for the associated federatedCommunity-SubList?

On my opinion the graphQL-Model have to define a GraphQL-Community-Typ containing attributes in the order of
1. with general attributs from both types of database-communities like:
- foreign
- publicKey
- url
2. with attributes only from the `federatedCommunities`-entries of this publicKey
   * several apiVersions should be read as a csv
3. then attributes only from the `communities`-entry of this publicKey

Reply:
Why there is not a community entry for every community?
In my opinion this strange behavior is the main cause of my trouble.

Reply:
So ich habe jetzt ein eigenes Model angelegt namens AdminCommunityView wo ich meinen Code den ich vorher im Community Model ergänzt habe reingetan habe. Somit ist das jetzt getrennt vom Standard Community Model.
Ich weiß nicht genau was du mir mit deinem Text sagen willst. Mein Code funktioniert, wo ist jetzt genau das Problem?
Wie soll mir das lesen der apiVersions als csv von Nutzen sein? Ich kann dir nicht folgen.